app/models/edition/role_appointments.rb

```diff
@@ -21,4 +21,11 @@ def can_be_associated_with_role_appointments?
   def search_index
     super.merge("people" => role_appointments.map(&:slug))
   end
+
+  def is_associated_with_a_minister?
```

Comment:
For safety, it might make sense to also check with `super` in case the overridden method elsewhere returns true when this one returns false. e.g.

```ruby

def is_associated_with_a_minister?
 has_ministerial_role_appointments? || super
end

def has_ministerial_role_appointments?
  role_appointments.any? do |role_appointment|
     role_appointment.role.is_a?(MinisterialRole)
  end
end
```

It doesn't look like any edition classes mix-in more than one module that override the `is_associated_with_a_minister?` method, but that could easily change in the future.